MobileCenterCrashes/MobileCenterCrashes/Internals/Utils/MSErrorLogFormatter.m

```diff
@@ -297,8 +297,8 @@ + (MSErrorReport *)errorReportFromLog:(MSAppleErrorLog *)errorLog {
   //the moment the app was launched and when the app crashed.
 
   NSDate *appStartTime =
-  [NSDate dateWithTimeIntervalSince1970:([errorLog.toffset doubleValue] - [errorLog.appLaunchTOffset doubleValue])];
-  NSDate *appErrorTime = [NSDate dateWithTimeIntervalSince1970:[errorLog.toffset doubleValue]];
+  [NSDate dateWithTimeIntervalSince1970:(([errorLog.toffset doubleValue]/1000) - ([errorLog.appLaunchTOffset doubleValue])/1000)];
```

Comment:
You can do it like this. 😜 
`A/1000 - B/1000` ==`(A - B) / 1000`
One less division.